pkgs/games/anki/default.nix

```diff
@@ -42,7 +42,7 @@ let
     pname = "anki-manual";
     inherit version;
     src = fetchFromGitHub {
-      owner = "dae";
+      owner = "ankitects";
```

Comment:
That’s not a no-op holy <s>f</s> heck.

I do hope that slipped into this commit by mistake.

Reply:
This being said, it looks like ankisrs.net now points to the ankitects repo, so IMO this is not a problem (even though it doesn't match the commit name). (Also, the build will probably not be affected, as the reference and hash stay the same)

Reply:
https://github.com/dae/ankidocs is a URL redirect to https://github.com/ankitects/anki-docs now, so it is a no-op.  I can change the `sha256` to garbage and nix will re-fetch and tell me the same hash.  Am I missing something?

Reply:
Yeah, please put it in a different commit explaining that.

Reply:
Done